lib/formatOrder.js

```diff
@@ -0,0 +1,16 @@
+var sorting = require('postcss-sorting');
+
+function formatOrder(root, params) {
+  var sortOrder = params.stylelint['declaration-block-properties-order'];
+  if (!Array.isArray(sortOrder)) {
```

Comment:
FYI, stylelint [does support property groups](https://github.com/stylelint/stylelint/tree/master/src/rules/declaration-block-properties-order#array-of-unprefixed-property-names-or-group-objects).